srcpkgs/libva/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'libva'
 pkgname=libva
-version=2.9.1
-revision=2
+version=2.10.0
```

Comment:
Update libva-glx as well?

Reply:
And leave a comment here about keeping these two up to date with each other.

Reply:
Thanks, I thought this was previously added.